CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add ability for custom tiptap extensions to access the options passed to rich text widgets at the area level
+
 ## 3.55.0
 
 ### Adds
 
-* Add `publicApiCheckAsync` wrapper method (and use it internally) to allow for async permission checks of REST APIs. This feature doesn't introduce any breaking changes.
-* Add ability to add custom rich-text-widget options per area.
+* Add `publicApiCheckAsync` wrapper method (and use it internally) to allow for overrides to do async permission checks of REST APIs. This feature doesn't introduce any breaking changes because the default implementation still invokes `publicApiCheck` in case developers have overridden it.
```

Comment:
Why is this in this PR? 🤔

Reply:
The PR was opened before the last release date so there ended up being a merge conflict in the CHANGELOG.